### DISCUSSION TITLE
Two attributes: "prices_today" and "prices"?

### DISCUSSION THREAD

**Discussion by home409ca** [PiotrMachowski/Home-Assistant-custom-components-TGE#5] [Q&A] [answered]
What is the difference between the attributes "prices_today" and "prices"?
They have the same values...

**Comment by KrzysztofHajdamowicz** [comment, marked as answer]
Prices_today show prices for current day, prices_tomorrow shows prices for next day (published around 17:00).
Prices aggregate that dictionaries.
